deck.gl/examples/scripting/interactivity.html

```diff
@@ -1,47 +1,14 @@
 <html>
   <head>
-    <!-- FONT -->
-    <link
-      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap"
-      rel="stylesheet"
-    />
-
-    <!-- deck.gl standalone bundle -->
     <script src="https://unpkg.com/deck.gl@^8.3.0/dist.min.js"></script>
-
-    <!-- @deck.gl/carto module -->
-    <script src="https://unpkg.com/@deck.gl/carto@latest/dist.min.js"></script>
-
-    <!-- Mapbox dependencies -->
-    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.js"></script>
-    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.css" rel="stylesheet" />
-
-    <style type="text/css">
-      body {
-        margin: 0;
-        padding: 0;
-      }
-      #map {
-        width: 100vw;
-        height: 100vh;
-      }
-      .deck-tooltip::after {
```

Comment:
It's a pity that we loose this, but I see it's in the interest of brevity, so ok